TODO/write-clean-css-10-simple-steps-pt1.md

```diff
@@ -1,131 +1,133 @@
 > * 原文地址：[How to Write Clean CSS in 10 Simple Steps Pt1](http://blog.alexdevero.com/write-clean-css-10-simple-steps-pt1/)
 * 原文作者：[Alex Devero](http://blog.alexdevero.com/)
 * 译文出自：[掘金翻译计划](https://github.com/xitu/gold-miner)
-* 译者：[]()
-* 校对者：[]()
+* 译者：[王子建](https://github.com/Romeo0906)
+* 校对者：[Mark](https://github.com/marcmoore)、[Tina92](https://github.com/Tina92)
 
-# How to Write Clean CSS in 10 Simple Steps Pt1
+# 如何书写整洁的 CSS 代码？只需十步！
 
 [![How to Write Clean CSS in 10 Simple Steps Pt1](https://i2.wp.com/blog.alexdevero.com/wp-content/uploads/2016/12/How-to-Write-Clean-CSS-in-10-Simple-Steps-Pt1-small.jpg?resize=697%2C464)](https://i2.wp.com/blog.alexdevero.com/wp-content/uploads/2016/12/How-to-Write-Clean-CSS-in-10-Simple-Steps-Pt1-small.jpg)
 
-Do you think you can write clean CSS? There are many web designers and developers convinced that writing good and clean CSS is hard. In a fact, a lot of people don’t like CSS at all. For them, CSS is pain in the butt. Here is a good news. Working with CSS can be actually easy! You can even enjoy it. This mini series, will help you with it. You will learn about 10 simple steps that will help you write cleaner CSS. And, clean CSS is one of the keys to CSS that is easy and pleasant to work with.
+你觉得自己可以写出整洁的 CSS 代码吗？许多网站的设计者和开发者都认为编写良好、整洁的 CSS 代码很难。事实上，许多人根本就不喜欢 CSS，对他们来说，CSS 是一个痛点。好消息来了，使用 CSS 实际上可以变得很简单，你甚至可以乐在其中。这个迷你系列的文章可以助你实现这个目的，只需十步即可得到整洁的 CSS 代码。要知道，书写整洁的 CSS 代码是简单愉快地使用 CSS 的关键之一呢！
 
-## Table of Contents:
+##　内容列表：
 
-Brief intro
+简介
 
-No.1: Stick to one naming convention
+No.1: 坚持一种命名规范
 
-Good and meaningful CSS classes
+良好的有意义的 CSS 类名
 
-The beauty of semantic naming convention
+语义化命名规范之美
 
-No.2: Use external stylesheets
+No.2: 使用外链样式表文件
 
-No.3: Validate your CSS
+No.3: 验证 CSS 代码
 
-Baptism by fire
+火的淬炼
 
-No.4: Use CSS linter
+No.4: 使用 CSS 代码 linter
 
-No.5: Adopt modular CSS
+No.5: 采用模块化的 CSS 代码
 
-Which modular framework to choose
+如何选择模块化框架
 
-No.6-10 in [part 2](http://blog.alexdevero.com/write-clean-css-10-simple-steps-pt2/).
+No.6-10 在[第二部分]() 
```

Comment:
the part 2 is [here](https://github.com/xitu/gold-miner/blob/master/TODO/write-clean-css-10-simple-steps-pt2.md)